### PR TITLE
Add Deletion button to archived runs 

### PR DESCRIPTION
**Pull request by jingzhang36**
Context 
https://github.com/kubeflow/pipelines/issues/3283

video https://drive.google.com/file/d/1ruipA6DBIa9QERE9ur5ZGCvUj_Xctr_w/view

This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/3285)

### CODE REVIEW

File: frontend/src/pages/Archive.test.tsx [hunk 1]
```diff
@@ -60,17 +67,29 @@ describe('Archive', () => {
     expect(updateBannerSpy).toHaveBeenCalledWith({});
   });
 
-  it('only enables restore button when at least one run is selected', () => {
+  it('only enables restore and delete button when at least one run is selected', () => {
```
Comment (Bobgy) [inline frontend/src/pages/Archive.test.tsx]:
```suggestion
  it('enables restore and delete button when at least one run is selected', () => {
```

File: frontend/src/pages/Archive.test.tsx [hunk 2]
```diff
@@ -85,4 +104,65 @@ describe('Archive', () => {
...
+    const confirmBtn = call.buttons.find((b: any) => b.text === 'Delete');
+    await confirmBtn.onClick();
+    await deleteRunSpy;
+    await TestUtils.flushPromises();
+    tree.update();
+    expect(deleteRunSpy).toHaveBeenCalledTimes(3);
+    expect(deleteRunSpy).toHaveBeenCalledWith('id1');
+    expect(deleteRunSpy).toHaveBeenCalledWith('id2');
+    expect(deleteRunSpy).toHaveBeenCalledWith('id3');
+    console.log(tree.state('selectedIds'));
```
Comment (Bobgy) [inline frontend/src/pages/Archive.test.tsx]: nit: forget to delete?

File: frontend/src/pages/Archive.test.tsx [hunk 3]
```diff
@@ -85,4 +104,65 @@ describe('Archive', () => {
...
+    const call = updateDialogSpy.mock.calls[0][0];
+    const cancelBtn = call.buttons.find((b: any) => b.text === 'Cancel');
+    await cancelBtn.onClick();
+    expect(deleteRunSpy).not.toHaveBeenCalled();
+  });
+
+  it('deletes selected ids when Confirm is clicked', async () => {
+    tree = shallow(<Archive {...generateProps()} />);
+    tree.setState({ selectedIds: ['id1', 'id2', 'id3'] });
+    console.log(tree.state('selectedIds'));
```
Comment (Bobgy) [inline frontend/src/pages/Archive.test.tsx]: nit: forget to delete?

**Comment by Bobgy** [comment]
FYI, your mentioned context in https://github.com/kubeflow/pipelines/pull/3285#issue-389140409 seems wrong. Could you double check?

File: frontend/src/pages/Archive.test.tsx (hunk 1 above)
Comment (jingzhang36) [reply frontend/src/pages/Archive.test.tsx]: done

File: frontend/src/pages/Archive.test.tsx (hunk 3 above)
Comment (jingzhang36) [reply frontend/src/pages/Archive.test.tsx]: yeah... done

File: frontend/src/pages/Archive.test.tsx (hunk 2 above)
Comment (jingzhang36) [reply frontend/src/pages/Archive.test.tsx]: done

**Comment by jingzhang36** [comment]
> FYI, your mentioned context in [#3285 (comment)](https://github.com/kubeflow/pipelines/pull/3285#issue-389140409) seems wrong. Could you double check?

yes. changed to the bug #3283

**Pull request merged**